catalog/dags/popularity/refresh_popularity_metrics_task_factory.py

```diff
@@ -63,22 +62,35 @@ def create_refresh_popularity_metrics_task_group(
                 "media_type": media_type,
                 "dag_id": refresh_config.dag_id,
                 "message": "Popularity metrics update complete | "
-                "_Next: popularity constants view update_",
+                "_Next: popularity constants update_",
             },
         )
 
-        update_constants = PythonOperator(
-            task_id=UPDATE_MEDIA_POPULARITY_CONSTANTS_TASK_ID,
-            python_callable=sql.update_media_popularity_constants,
-            op_kwargs={
-                "postgres_conn_id": POSTGRES_CONN_ID,
-                "media_type": media_type,
-            },
-            execution_timeout=execution_timeout,
-            doc=(
-                "Updates the popularity constants view. This completely "
-                "recalculates the popularity constants for each provider."
-            ),
+        # For each provider that supports popularity data for this media type,
+        # recalculate the percentile value used to generate the constant.
+        update_constants = (
+            sql.update_percentile_and_constants_for_provider.override(
+                group_id=UPDATE_MEDIA_POPULARITY_CONSTANTS_TASK_ID,
+            )
+            .partial(
+                postgres_conn_id=POSTGRES_CONN_ID,
+                media_type=media_type,
+                execution_timeout=execution_timeout,
+            )
+            .expand(
+                provider=[
+                    provider
+                    for provider in sql.POPULARITY_METRICS_BY_MEDIA_TYPE[
+                        media_type
+                    ].keys()
+                ]
+            )
+        )
+        update_constants.doc = (
+            "Recalculate the percentile values and popularity constants"
+            " for each provider, and update them in the metrics table. The"
+            " popularity constants will be used to calculate standardized"
+            " popularity scores."
         )
```

Comment:
Is the comment necessary with the doc string? The comment feels like it duplicates the task's doc string :thinking: